File: zadania wg wideo/zad.3/app/src/main/java/com/example/zadanie3/MainActivity.java
package com.example.zadanie3;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Zad.1:
        // Log.d("test","output");

        //Zad.4:
        //Log.d("fail","it's not working");

        //Zad.11:
        //findViewById(R.id.button5);

    }

    //Zad.2:
    public void disable(View v){
        /*v.setEnabled(false);

        //Zad.3:
        //Log.d("Success","Button Disabled");

        //Zad.5:
        //Button button = (Button) v;
        .setText("Disabled");
         */

        /* //Zad.6:
        View myView = findViewById(R.id.button0);
        myView.setEnabled(false);
        Button button = (Button) myView;
        button.setText("New Disabled");
         */

        //Zad.7:
        //findViewById(R.id.button0).setEnabled(false);
        //((Button)findViewById(R.id.button0)).setText("new new Disabled");
    }

    public void handleText(View v){
        /*
        //Zad.8:
        TextView t = findViewById(R.id.source);
        String input = t.getText().toString();

        Log.d("info",input); */

        /*
        //Zad.9:
        TextView t = findViewById(R.id.source);
        String input = t.getText().toString();
        ((TextView)findViewById(R.id.output)).setText(input);
        Log.d("info",input); */

        /*
        //Zad.10:
        EditText t = findViewById(R.id.source);
        String input = t.getText().toString();
        ((TextView)findViewById(R.id.output)).setText(input);
        //Toast.makeText(this,"Alert",Toast.LENGTH_LONG).show();
        //Log.d("info",input);
        Toast.makeText(this,input,Toast.LENGTH_LONG).show(); */

        /*
        //Zad.12:
        v.setEnabled(false);
        Log.d("muLog","message");
        Toast.makeText(this, "clicked",
                Toast.LENGTH_LONG).show();
         */
    }

    //Zad.13
    public void launchSettings(View v){
        //launch a new activity
        Intent i = new Intent(this,launchSettings.class);
        startActivity(i);
    }
}

File: Praca z DB | SQLite/app/src/main/java/com/example/sqlite/MainActivity.kt
package com.example.sqlite

import android.app.Activity
import android.app.AlertDialog
import android.database.sqlite.SQLiteDatabase
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText


class MainActivity() : Activity(), View.OnClickListener {
    var Rollno: EditText? = null
    var Name: EditText? = null
    var Marks: EditText? = null
    var Insert: Button? = null
    var Delete: Button? = null
    var Update: Button? = null
    var View: Button? = null
    var ViewAll: Button? = null
    var db: SQLiteDatabase? = null

    /** Called when the activity is first created.  */
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Rollno = findViewById<View>(R.id.Rollno) as EditText
        Name = findViewById<View>(R.id.Name) as EditText
        Marks = findViewById<View>(R.id.Marks) as EditText
        Insert = findViewById<View>(R.id.Insert) as Button
        Delete = findViewById<View>(R.id.Delete) as Button
        Update = findViewById<View>(R.id.Update) as Button
        View = findViewById<View>(R.id.View) as Button
        ViewAll = findViewById<View>(R.id.ViewAll) as Button
        Insert!!.setOnClickListener(this)
        Delete!!.setOnClickListener(this)
        Update!!.setOnClickListener(this)
        View!!.setOnClickListener(this)
        ViewAll!!.setOnClickListener(this)

        // Creating database and table
        db = openOrCreateDatabase("StudentDB", MODE_PRIVATE, null)
        db.execSQL("CREATE TABLE IF NOT EXISTS student(rollno VARCHAR,name VARCHAR,marks VARCHAR);")
    }

    override fun onClick(view: View) {
        // Inserting a record to the Student table
        if (view === Insert) {
            // Checking for empty fields
            if (Rollno!!.text.toString().trim { it <= ' ' }.length == 0 || Name!!.text.toString()
                    .trim { it <= ' ' }.length == 0 || Marks!!.text.toString()
                    .trim { it <= ' ' }.length == 0
            ) {
                showMessage("Error", "Please enter all values")
                return
            }
            db!!.execSQL(
                "INSERT INTO student VALUES('" + Rollno!!.text + "','" + Name!!.text +
                        "','" + Marks!!.text + "');"
            )
            showMessage("Success", "Record added")
            clearText()
        }
        // Deleting a record from the Student table
        if (view === Delete) {
            // Checking for empty roll number
            if (Rollno!!.text.toString().trim { it <= ' ' }.length == 0) {
                showMessage("Error", "Please enter Rollno")
                return
            }
            val c =
                db!!.rawQuery("SELECT * FROM student WHERE rollno='" + Rollno!!.text + "'", null)
            if (c.moveToFirst()) {
                db!!.execSQL("DELETE FROM student WHERE rollno='" + Rollno!!.text + "'")
                showMessage("Success", "Record Deleted")
            } else {
                showMessage("Error", "Invalid Rollno")
            }
            clearText()
        }
        // Updating a record in the Student table
        if (view === Update) {
            // Checking for empty roll number
            if (Rollno!!.text.toString().trim { it <= ' ' }.length == 0) {
                showMessage("Error", "Please enter Rollno")
                return
            }
            val c =
                db!!.rawQuery("SELECT * FROM student WHERE rollno='" + Rollno!!.text + "'", null)
            if (c.moveToFirst()) {
                db!!.execSQL(
                    ("UPDATE student SET name='" + Name!!.text + "',marks='" + Marks!!.text +
                            "' WHERE rollno='" + Rollno!!.text + "'")
                )
                showMessage("Success", "Record Modified")
            } else {
                showMessage("Error", "Invalid Rollno")
            }
            clearText()
        }
        // Display a record from the Student table
        if (view === View) {
            // Checking for empty roll number
            if (Rollno!!.text.toString().trim { it <= ' ' }.length == 0) {
                showMessage("Error", "Please enter Rollno")
                return
            }
            val c =
                db!!.rawQuery("SELECT * FROM student WHERE rollno='" + Rollno!!.text + "'", null)
            if (c.moveToFirst()) {
                Name!!.setText(c.getString(1))
                Marks!!.setText(c.getString(2))
            } else {
                showMessage("Error", "Invalid Rollno")
                clearText()
            }
        }
        // Displaying all the records
        if (view === ViewAll) {
            val c = db!!.rawQuery("SELECT * FROM student", null)
            if (c.count == 0) {
                showMessage("Error", "No records found")
                return
            }
            val buffer = StringBuffer()
            while (c.moveToNext()) {
                buffer.append("Rollno: " + c.getString(0) + "\n")
                buffer.append("Name: " + c.getString(1) + "\n")
                buffer.append("Marks: " + c.getString(2) + "\n\n")
            }
            showMessage("Student Details", buffer.toString())
        }
    }

    fun showMessage(title: String?, message: String?) {
        val builder = AlertDialog.Builder(this)
        builder.setCancelable(true)
        builder.setTitle(title)
        builder.setMessage(message)
        builder.show()
    }

    fun clearText() {
        Rollno!!.setText("")
        Name!!.setText("")
        Marks!!.setText("")
        Rollno!!.requestFocus()
    }
}